content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -30,9 +30,9 @@ mutating and validating (respectively)
 which are configured in the API.
 
 Admission controllers may be "validating", "mutating", or both. Mutating
-controllers may modify the objects they admit; validating controllers may not.
+controllers may modify the requests they admit; validating controllers may not.
```

Comment:
They don't modify the requests, to the best of my knowledge. How about rephrasing the line as:

> Mutating
controllers may modify the request objects they admit; validating controllers may not.

Reply:
I got your point.
But `request objects` also confuses me.